Francisco_Trujillo/Some Function/anyfunction.js

```diff
@@ -0,0 +1,58 @@
+function runningLogger(){
+    console.log("I am running");
+}
+
+runningLogger();
+
+function multiplyByTen(num){
+    console.log(num*10);
+}
+
+multiplyByTen(20);
+
+function stringReturnOne(){
+    
+    return 'One';
+}
+
+function stringReturnTwo(){
+    return 'two';
+    
+}
+
+console.log(stringReturnOne(), stringReturnTwo());
+
+function caller(para){
+    if(typeof(para) == 'function' ){
+        para;
+    }
+        
+}
+
+caller(runningLogger());
+
+function myDoubleConsoleLog(par1, par2){
```

Comment:
Try to use more significantly named parameters. What is par1? function1 might be more so helpful.